db/migrate/20191128111630_add_restart_needed_to_vms.rb

```diff
@@ -0,0 +1,5 @@
+class AddRestartNeededToVms < ActiveRecord::Migration[5.1]
+  def change
+    add_column :vms, :restart_needed, :boolean, :null => false, :default => false
```

Comment:
We should drop the `:null => false` because most providers won't set this, also we should drop the `:default => false`, we try to keep the "logic" at the rails not the db level and if it is nil it will behave falsy anyway `if nil => false`

Reply:
Yes, absolutely, I was used to that to avoid the three state problem. I'll modify that and re-push

Reply:
Yeah actually personally I prefer having db-level defaults and no nils and we do that in other projects (topological-inventory for example) but in MIQ we avoid db-level defaults